components/frontend/eslint.config.mjs

```diff
@@ -0,0 +1,63 @@
+import { fixupConfigRules } from "@eslint/compat"
+import pluginJs from "@eslint/js"
+import pluginJest from "eslint-plugin-jest"
+import pluginNode from "eslint-plugin-n"
+import pluginPrettierConfigRecommended from "eslint-plugin-prettier/recommended"
+import pluginPromise from "eslint-plugin-promise"
+import pluginReactJSXRuntime from "eslint-plugin-react/configs/jsx-runtime.js"
+import pluginReactConfigRecommended from "eslint-plugin-react/configs/recommended.js"
+import pluginSimpleImportSort from "eslint-plugin-simple-import-sort"
+import globals from "globals"
+
+export default [
+    pluginJs.configs.recommended,
+    pluginJest.configs["flat/recommended"],
+    pluginNode.configs["flat/recommended-module"],
+    ...fixupConfigRules(pluginReactConfigRecommended),
+    ...fixupConfigRules(pluginReactJSXRuntime),
+    pluginPrettierConfigRecommended,
+    {
+        plugins: {
+            "simple-import-sort": pluginSimpleImportSort,
+            promise: pluginPromise,
+        },
+        languageOptions: {
+            globals: { ...globals.browser, ...globals.node },
+            parserOptions: {
+                ecmaFeatures: {
+                    jsx: true,
+                },
+            },
+        },
+        rules: {
+            "no-unused-vars": ["error", { argsIgnorePattern: "^_" }],
+            "n/no-unsupported-features/node-builtins": "off",
```

Comment:
What issues do we encounter if we toggle this?

Reply:
```console
/Users/fniessink/Developer/quality-time/components/frontend/src/api/fetch_server_api.js
  17:12  error  The 'fetch' is still an experimental feature and is not supported until Node.js 21.0.0. The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins

/Users/fniessink/Developer/quality-time/components/frontend/src/header_footer/DownloadAsPDFButton.js
  21:27  error  The 'URL.createObjectURL' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins

/Users/fniessink/Developer/quality-time/components/frontend/src/header_footer/DownloadAsPDFButton.test.js
  67:5  error  The 'URL.createObjectURL' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins

/Users/fniessink/Developer/quality-time/components/frontend/src/widgets/Button.js
  368:9   error  The 'navigator' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins
  375:21  error  The 'navigator' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins

/Users/fniessink/Developer/quality-time/components/frontend/src/widgets/Button.test.js
  335:19  error  The 'navigator' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins
  342:12  error  The 'navigator' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins
  347:19  error  The 'navigator' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins
  359:19  error  The 'navigator' is still an experimental feature The configured version range is '>=20.0.0'  n/no-unsupported-features/node-builtins
```